src/main.rs

```diff
@@ -58,6 +58,45 @@ fn main() {
             SubCommand::with_name("list")
                 .alias("l")
                 .about("Lists the exercises available in rustlings")
+                .arg(
+                    Arg::with_name("paths")
```

Comment:
Personal opinion: I don't see much value in adding format-controlling options. If a user wants to extract a particular column, they could probably figure out some shell hackery to do so (e.g. `cut`).

The same almost applies for the filtering options (they could use `grep`), except perhaps `filter` if we wanted to save some disk reads (probably overkill).

Reply:
I personally don't want to assume that someone has `grep`, `cut`, `sed`, or `awk` on their machine to split and extract fields, so I think that the filtering adds quite a bit of value here. Controlling the displayed fields allows for the _shell utility use cases_ I mentioned in the PR. Otherwise, a user would have to skip the header row first (with `head` or `tail` which may not be available on a user's machine) before going about extracting and filtering on fields.
I would prefer to keep these features in, but I am happen to edit/update for better UX.

Reply:
I think that is safe to assume that there are _some_ CLI tools in the user's operating system that would allow for similar capabilities, _and_ that the use cases you showed above already presuppose a good deal of familiarity with CLI tools.

But as I said, this is a matter of taste, I don't really have a strong opinion :+1:

Reply:
Let's leave it as is for now. It's one file update, so it should be easy to retract if it ends up being a nightmare of a feature.